content/_index.md

```diff
@@ -263,6 +263,7 @@ Here are some that we like:
 * Node: [oclif](https://oclif.io/)
 * Python: [Click](https://click.palletsprojects.com/), [Typer](https://github.com/tiangolo/typer)
 * Ruby: [TTY](https://ttytoolkit.org/)
+* Rus: [structopt](https://github.com/TeXitoi/structopt), [clap](https://github.com/clap-rs/clap)
```

Comment:
Rust is misspelled here.

Also, could you please put clap first (since structopt appears to extend it),
and link to the [website for clap](https://clap.rs/) instead of the GitHub?

Reply:
Sure thing. I didn't use the clap website because I was getting HTTPS warnings, but I'll restore it.

Reply:
Additionally, I wasn't sure if structopt (being the easier to use of them) made sense as the first one to point people towards. Changed.